### PR TITLE
Add `$isDraft` function unit tests

### PR DESCRIPTION
**Pull request by shay2025**
## Description

This pull request introduces `$isDraft` unit tests.

## Related issue

*None*

## Type of change

Improvements (non-breaking change without functionality)

## How was this tested?

Ran `task test` command.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have ran `task check -f` and have no issues

### CODE REVIEW

**Comment by reviewpad-team** [comment]
**Reviewpad Report**

**:information_source: Messages**
* If you are adding or updating a built-in, please do not forget to update the [docs](https://github.com/reviewpad/docs)
* If you have added a new function or action do not forget to include it in the built-in list!

**Comment by reviewpad-team** [comment]
**Reviewpad Report**

**:information_source: Messages**
* If you are adding or updating a built-in, please do not forget to update the [docs](https://github.com/reviewpad/docs)
* If you have added a new function or action do not forget to include it in the built-in list!

File: plugins/aladino/functions/isDraft_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"github.com/google/go-github/v42/github"
+	"github.com/reviewpad/reviewpad/v2/lang/aladino"
+	mocks_aladino "github.com/reviewpad/reviewpad/v2/mocks/aladino"
+	plugins_aladino "github.com/reviewpad/reviewpad/v2/plugins/aladino"
+	"github.com/stretchr/testify/assert"
+)
+
+var isDraft = plugins_aladino.PluginBuiltIns().Functions["isDraft"].Code
+
+func TestIsDraftOnNilPullRequest(t *testing.T) {
```
Comment (ferreiratiago) [inline plugins/aladino/functions/isDraft_test.go:20]: I'm not sure if this is a use case. I know that we have a condition at `isDraft` with `if pullRequest == nil` but I'm not sure if this is a possibility. Can you please check this?

Also, if this is a possibility then we need to update our builtin in order to make sure that this case is cover (e.g. `description`).

File: plugins/aladino/functions/isDraft_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	assert.EqualError(t, err, "isDraft: pull request is nil")
+}
+
+func TestIsDraft_WhenTrue(t *testing.T) {
+    mockedEnv, err := mocks_aladino.MockDefaultEnv()
+	if err != nil {
+		log.Fatalf("mockDefaultEnv failed: %v", err)
+	}
+
+    mockedEnv.GetPullRequest().Draft = github.Bool(true)
```
Comment (ferreiratiago) [inline plugins/aladino/functions/isDraft_test.go:51]: This is not the correct way. We should use `GetDefaultMockPullRequestDetailsWith`.

```go
mockedPullRequest := mocks_aladino.GetDefaultMockPullRequestDetailsWith(&github.PullRequest{
		Draft: github.Bool(true),
	})
```

File: plugins/aladino/functions/isDraft_test.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	assert.Equal(t, wantVal, gotVal, "pull request should be in draft")
+}
+
+func TestIsDraft_WhenFalse(t *testing.T) {
+    mockedEnv, err := mocks_aladino.MockDefaultEnv()
+	if err != nil {
+		log.Fatalf("mockDefaultEnv failed: %v", err)
+	}
+
+    mockedEnv.GetPullRequest().Draft = github.Bool(false)
```
Comment (ferreiratiago) [inline plugins/aladino/functions/isDraft_test.go:68]: The same here.

**Review by ferreiratiago — CHANGES_REQUESTED** [review]
Please have a look at the comments.

**Review by ferreiratiago — APPROVED** [review]

**Pull request merged**